### ISSUE TITLE
Image Plugin - Uncaught TypeError: e is undefined - AND Solution

### ISSUE THREAD

**Issue by JugglingCoder** [tinymce/tinymce#7338]
Ver: 5.9.2

Bug Description: on clicking "Save" in the Image Upload dialog box, an ERROR is thrown by javaScript "Uncaught TypeError: e is undefined" for the image/plugin.min.js file.

Steps to Reproduce: 
1. in "tinymce.init" add the "plugins" parameter and add the "image" plugin. example - plugins: 'image',
2. in "tinymce.init" add the "valid_elements" parameter with very few elements, example -  valid_elements : 'p[style],h1[style]',
3. That's it.  Now try to add an image.  Even if your back-end processing is set up correctly (the image uploads, in the right folder, etc.), when you try to click "Save" on the Image Upload dialog box javaScript will throw an ERROR.

Expected Behavior: Image should be placed in WYSIWYG area.

!! SOLUTION !!
After 12+ hours of debugging this... checking, re-checking... line by line stuff, I figured out the "issue" and this should definitely be put into TinyMCE docs for setup.  My images were uploading fine, were in the correct folders and the XMLHttpRequest data was being sent fine.  Finally... after tons of "googling" "tinymce e is undefined" (not much there) I found a 10 year old StackOverflow reply to the effect,
"... it's a tinymce bug. Internally, the tinymce code uses a <span id="mce_marker"></span> to remember the caret-position when pasting."
And then it hit me... in my "tinymce.init" I had added the "valid_elements" parameter.  I thought, "maybe the valid_elements parameter is not allowing TinyMCE to place the tags it needs to add the image to the page?"  So, I commented the "valid_elements" param out... and it WORKED.

So, in summary, If using the Image Plugin and the "valid_elements" parameter,  I'm pretty sure you must allow certain elements for it to work correctly.  This isn't so much a bug as is it needs to be in the Docs for the Image Plugin setup... I think?  it would be nice to know exactly what elements (an example) must be allowed in order for it to work correctly.
